### PR TITLE
chore(deps): bump ember-template-lint to v5.3.0

### PR DESCRIPTION
**Pull request by luytena**

### CODE REVIEW

File: .template-lintrc.js [hunk 1]
```diff
@@ -5,5 +5,12 @@ module.exports = {
   rules: {
     "no-bare-strings": true,
     "no-implicit-this": { allow: ["housing-stat-link", "current-quarter"] },
```
Comment (anehx) [inline .template-lintrc.js:7]: Could you check why this is allowed? :thinking:

**Review by anehx — COMMENTED** [review]
LGTM otherwise

File: .template-lintrc.js (hunk 1 above)
Comment (luytena) [reply .template-lintrc.js:7]: The linter fails for the housing-stat-link helper:
```
addon/components/housing-stat-link.hbs
  3:9  error  Ambiguous path 'housing-stat-link' is not allowed. Use '@housing-stat-link' if it is a name
d argument or 'this.housing-stat-link' if it is a property on 'this'. If it is a helper or component that
 has no arguments, you must either convert it to an angle bracket invocation or manually add it to the 'n
o-implicit-this' rule configuration, e.g. 'no-implicit-this': { allow: ['housing-stat-link'] }.  no-implicit-this
```
```hbs
<a
  class="uk-button uk-button-default"
  href={{housing-stat-link}}
  target="_blank"
  rel="noopener noreferrer"
>
  {{t "ember-gwr.nav.housingStatLink"}}
</a>
```

Comment (anehx) [reply .template-lintrc.js:7]: Ah damn.

**Review by anehx — APPROVED** [review]

**Pull request merged**